Content.Client/Cargo/UI/BountyHistoryEntry.xaml

```diff
@@ -0,0 +1,26 @@
+﻿<BoxContainer xmlns="https://spacestation14.io"
+                xmlns:customControls="clr-namespace:Content.Client.Administration.UI.CustomControls"
+                Margin="10 10 10 0"
+                HorizontalExpand="True"
+                Visible="True">
+    <PanelContainer StyleClasses="AngleRect" HorizontalExpand="True">
+        <BoxContainer Orientation="Vertical"
+                      HorizontalExpand="True">
+            <BoxContainer Orientation="Horizontal">
+                <BoxContainer Orientation="Vertical" HorizontalExpand="True">
+                    <RichTextLabel Name="RewardLabel"/>
+                    <RichTextLabel Name="ManifestLabel"/>
+                </BoxContainer>
+                <Control MinWidth="10"/>
```

Comment:
Has this been resolved? (Instead of padding control, just use a margin)

```suggestion
                <BoxContainer Orientation="Vertical" HorizontalExpand="True" Margin="0 0 10 0">
                    <RichTextLabel Name="RewardLabel" />
                    <RichTextLabel Name="ManifestLabel" />
                </BoxContainer>
```

Reply:
Yeah, missed that